docs/quickstart.html

```diff
@@ -176,7 +178,7 @@
 
 		
 		<section class="comment">
-			<p><em>This is the quick getting started guide. For in depth instructions, check out the <a href="/docs/standalone-lsg.html">Standalone Live Style Guide</a>.</em></p>
+			<p><em>This is the quick getting started guide. For in depth instructions, check out the <a href="/docs/standalone-lsg.html">Standalone Living Style Guide</a>.</em></p>
```

Comment:
This link doesn't go anywhere. We may want to rephrase for your new structure like "This is the quick start guide. For in-depth set up instructions, check out the <a>set up section</a>